recipes-core/images/console-image.bb

```diff
@@ -1,6 +1,6 @@
-SUMMARY = "A console development image with some C/C++ dev tools"
+SUMMARY = "A console enabled image with minimal additions to run all of Pelion"
 
-IMAGE_FEATURES[validitems] += "tools-debug tools-sdk"
+IMAGE_FEATURES[validitems] += ""
```

Comment:
I suppose this line could be removed if no valid items are added. I suppose there needs to be a separate, a bit heavier development image where these  could be added